components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/OperationPoliciesApiServiceImpl.java

```diff
@@ -312,4 +373,65 @@ public Response getCommonOperationPolicyContentByPolicyId(String operationPolicy
         }
         return null;
     }
+
+    /**
+     * Export Operation Policy as an archived file.
+     *
+     * @param name           Name of the operation policy
+     * @param version        Version of the operation policy
+     * @param messageContext message context
+     * @throws APIManagementException If an error occurs while creating the directory, transferring files or
+     *                                extracting the content
+     * @returnA zip file containing both (if exists) operation policy specification and policy definition
+     */
+    @Override
+    public Response exportOperationPolicy(String name, String version, String format, MessageContext messageContext)
+            throws APIManagementException {
+
+        APIProvider apiProvider = RestApiCommonUtil.getLoggedInUserProvider();
+        String organization = RestApiUtil.getValidatedOrganization(messageContext);
+
+        OperationPolicyData policyData = apiProvider.getCommonOperationPolicyByPolicyName(name, version, organization,
+                true);
+        if (policyData != null) {
+            File file = RestApiPublisherUtils.exportOperationPolicyData(policyData, format);
+            return Response.ok(file).header(RestApiConstants.HEADER_CONTENT_DISPOSITION,
+                    "attachment; filename=\"" + file.getName() + "\"").build();
+        } else {
+            // if not found, throw not found error
+            throw new APIManagementException(
+                    "Couldn't retrieve an existing common policy with Name: " + name + " and Version: " + version,
+                    ExceptionCodes.from(ExceptionCodes.OPERATION_POLICY_NOT_FOUND_WITH_NAME_AND_VERSION, name, version));
+        }
+    }
+
+    /**
+     * @param fileInputStream Archived file
+     * @param fileDetail      file details
+     * @param messageContext  message context
+     * @return return a response with the corresponding status code
+     * @throws APIManagementException If an error occurs while creating the directory, transferring files or
+     *                                extracting the content
+     */
+    @Override
+    public Response importOperationPolicy(InputStream fileInputStream, Attachment fileDetail,
+            MessageContext messageContext) throws APIManagementException {
+
+        APIProvider apiProvider = RestApiCommonUtil.getLoggedInUserProvider();
+        String organization = RestApiUtil.getValidatedOrganization(messageContext);
+        String extractedFolderPath = null;
+        OperationPolicyDataDTO createdPolicy = null;
+        URI createdPolicyUri = null;
+
+        try {
+            extractedFolderPath = ImportUtils.getArchivePathOfPolicyExtractedDirectory(fileInputStream);
+            createdPolicy = ImportUtils.importPolicy(extractedFolderPath, organization, apiProvider);
+            createdPolicyUri = new URI(RestApiConstants.REST_API_PUBLISHER_VERSION + "/"
```

Comment:
Can we use this URL in the browser?

Reply:
It should. Or else we can simply return the createdPolicy with ok response